lib/metadata/get-assignee.js

```diff
@@ -0,0 +1,17 @@
+"use strict";
+var github = require('../github');
+
+module.exports = function getAssignee(number, metadata) {
+    return github.get('/repos/:owner/:repo/issues/:number', { number: number }).then(function(issue) {
+      if (issue.assignee) {
+        return null;
```

Comment:
I understand the intent, but it may be confusing for others to find a module named "get-assignee" which returns `null` for pull requests that have an assignee. Renaming this to "select-assignee" (or "choose-assignee", etc.) would satisfy my concern; what do you think?

Reply:
Changed to `choose-assignee`